docs/content/learn/presentations/OSCAL-mini-workshop-series.md

```diff
@@ -0,0 +1,48 @@
+# OSCAL Mini Workshop Series
+
+The NIST [OSCAL](/) team is hosting a new series of mini workshops. that aim to address topics of interest for our community and to open this forum for its members to present their [OSCAL](/)-related work. Unless specifically stated, the workshops will not require a deep, technical understanding of [OSCAL](/), and the dialog is informal, allowing the community to interact with the presenters and with the [OSCAL](/) team members. 
```

Comment:
Minor nit: "that aim ..." is an incomplete sentence.

```suggestion
The NIST [OSCAL](/) team is hosting a new series of mini workshops. They aim to address topics of interest for our community and to open this forum for its members to present their [OSCAL](/)-related work. Unless specifically stated, the workshops will not require a deep, technical understanding of [OSCAL](/), and the dialog is informal, allowing the community to interact with the presenters and with the [OSCAL](/) team members. 
```

Reply:
The period in front of "that" was extra. I removed it and added the right tense. It reads now:
The NIST [OSCAL](/) team is hosting a new series of mini workshops that aims to address topics of interest for our community and to open this forum for its members to present their [OSCAL](/)-related work.

Reply:
In the interest of time, to avoid another round of reviews, I decided to leave it as suggested and not as listed above (and corrected locally)